shared/src/test/diff/qq/Nested.mls

```diff
@@ -109,4 +109,4 @@ fun f(x) = code"${run(x)} + 1"
 //│ fun f: forall 'a. Code[Code[Int, 'a], nothing] -> Code[Int, 'a]
 
 code"x => ${code"code"code"${x}"""}"
```

Comment:
Similar remark as above. Add a test with another variable than the top level x that's in scope.